_rules/non-visual-location-reference-825ule.md

```diff
@@ -0,0 +1,506 @@
+- [WCAG 2.1 - Understanding Success Criterion 1.3.3: Sensory Characteristics](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html)
+
+### Related WCAG Techniques
+
+- [G96: Providing textual identification of items that otherwise rely only on sensory information to be understood](https://www.w3.org/WAI/WCAG21/Techniques/general/G96)
+- [F14: due to identifying content only by its shape or location](https://www.w3.org/WAI/WCAG21/Techniques/failures/F14)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The content in the column on the right is indicated with the word "box" (which is a shape indicator based on visual perception) but also indicated by referencing the word "howdy".
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	button {
+		background: #FAA;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the "howdy" box on the right, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 2
+
+The content in the column on the right is indicated with the word "right" (which is a location indicator based on visual perception) but also indicated by referencing that the content can be found below this content in the DOM order.
+
+```html
+<head>
+<style>
+.col-container {
+  display: table;
+}
+.col {
+  display: table-cell;
+  padding: 16px;
+}
+</style>
+</head>
+<body>
+
+<div class="col-container">
+  <div class="col">
+    <p>Read the content in the right column (or below this paragraph) about the fuzzyness of cats</p>
+  </div>
+
+  <div class="col">
+    <p>Cats are usually fuzzy.</p>
+  </div>
+</div>
+
+</body>
+```
+
+#### Passed Example 3
+
+The content in the column on the right is a menu. The user is told to find the menu on the right (which is a visual indicator word) but the menu is also identified with the help of the "menu" heading above the content.
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	button {
+		background: #FAA;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
+		</div>
+		<div class="col">
+			<h1>Menu</h1>
+			<ul>
+				<li>
+					<a href="https://www.w3.org/Consortium/contact">Contact</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Help/">Help and FAQ</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/sponsor/">Donate</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/siteindex">Sitemap</a>
+				</li>
+			</ul>
+					
+		</div>
+	</div>
+</body>
+```
+
+
+#### Passed Example 4
+
+This document is using the word "square" but in this case it is no instruction.
```

Comment:
It is a visual indicator word according to the definition. So it is applicable. The expectation has mentions "except if the target is not part of an instruction about [web content](https://www.w3.org/TR/WCAG21/#dfn-content)". Which is what this example is about.

Reply:
Maybe elaborating "it is not part of an instruction (because it is not used in the meaning of  a shape)" or something like that.

Or add a note to the (exception of the) expectation in the line of "visual indicator may be homonym of something else in which case they are not instruction (_e.g._ below (down)/below (further), square (shape)/square (x²), …)".

I feel the important difference here is whether the "visual indicator" is a string that match a set of string (and expectation checks its meaning), or a specific meaning that happen to be spelled that way. In the first case, "square" is a visual indicator that may or may not be an instruction. In the second case, "square" may or may not be a visual indicator. Adding a note to the expectation will make it clear that we consider the first case.

Reply:
It does matches the set of string and doesn't share its meaning. But even if it did share the meaning (and an example can be thought of fairly easily) it should still not fail because it is not meant to be a reference to web content. That is the important bit I would say.

I have expanded the explanation of the example.